src/wallet/components/WalletDropdown.tsx

```diff
@@ -18,7 +38,12 @@ export function WalletDropdown({ children, className }: WalletDropdownReact) {
   } = useWalletContext();
 
   const childrenArray = useMemo(() => {
-    return Children.toArray(children).map((child) => {
+    // default children implementation
+    const childrenToClone = children
+      ? Children.toArray(children)
+      : defaultWalletDropdownChildren;
```

Comment:
note: Related to the previous comment, but here the children _will_ rerender since they're being cloned below and will therefore _not_ have the same reference.